### PR TITLE
[2D Game Tutorial - Creating the Enemy] Minor clarification on 3 lines

### PR DESCRIPTION
**Pull request by TheYellowArchitect**

### CODE REVIEW

File: getting_started/first_2d_game/04.creating_the_enemy.rst [hunk 1]
```diff
@@ -24,7 +24,7 @@ Click Scene -> New Scene and add the following nodes:
 Don't forget to set the children so they can't be selected, like you did with
 the Player scene.
 
-In the :ref:`RigidDynamicBody2D <class_RigidDynamicBody2D>` properties, set ``Gravity Scale``
+In the properties of :ref:`RigidDynamicBody2D <class_RigidDynamicBody2D>` (named ``Mob``), set ``Gravity Scale``
```
Comment (mhilbrunner) [inline getting_started/first_2d_game/04.creating_the_enemy.rst:27]:
```suggestion
In the properties of the :ref:`RigidDynamicBody2D <class_RigidDynamicBody2D>` (named ``Mob``), set ``Gravity Scale``
```

File: getting_started/first_2d_game/04.creating_the_enemy.rst [hunk 2]
```diff
@@ -150,14 +150,14 @@ selects a random integer between ``0`` and ``n-1``.
...
-Connect the ``screen_exited()`` signal of the ``VisibleOnScreenNotifier2D`` node and
+Connect the ``screen_exited()`` signal of the ``VisibleOnScreenNotifier2D`` node onto the ``Mob`` and
 add this code:
 
 .. tabs::
  .. code-tab:: gdscript GDScript
 
     func _on_visible_on_screen_notifier_2d_screen_exited():
-        queue_free()
+        queue_free() #Destroys this node in the next frame
```
Comment (mhilbrunner) [inline getting_started/first_2d_game/04.creating_the_enemy.rst:160]:
```suggestion
        queue_free() # Destroys this node in the next frame
```

File: getting_started/first_2d_game/04.creating_the_enemy.rst [hunk 3]
```diff
@@ -150,14 +150,14 @@ selects a random integer between ``0`` and ``n-1``.
             use ``randomize()`` in our ``Main`` scene, so we won't need it here.
 
 The last piece is to make the mobs delete themselves when they leave the screen.
-Connect the ``screen_exited()`` signal of the ``VisibleOnScreenNotifier2D`` node and
+Connect the ``screen_exited()`` signal of the ``VisibleOnScreenNotifier2D`` node onto the ``Mob`` and
```
Comment (mhilbrunner) [inline getting_started/first_2d_game/04.creating_the_enemy.rst:153]:
```suggestion
Connect the ``screen_exited()`` signal of the ``VisibleOnScreenNotifier2D`` node to the ``Mob`` and
```

**Review by mhilbrunner — CHANGES_REQUESTED** [review]
Nice! Some minor changes to improve consistency with other docs above.

**Comment by TheYellowArchitect** [comment]
I agree with all of them and made the changes, you have a good eye to catch those hahaha

**Comment by Vennnot** [comment]
This can now be merged, no?

**Comment by mhilbrunner** [comment]
I've salvaged a change from this in https://github.com/godotengine/godot-docs/pull/7372. For the others, I think they aren't necessary - three lines above the first it is pointed out very clearly that the `Mob` node is being referred to, and the `queue_free()` comment IMO isn't necessary.

Thanks for contributing!

**Comment by TheYellowArchitect** [comment]
I thank you :)

**Pull request closed without merging**